tests/testing_common.py

```diff
@@ -567,6 +569,34 @@ def _test_merge_layers(self, model_id, config_cls, config_kwargs):
         logits_merged_from_pretrained = model_from_pretrained(**dummy_input)[0]
         self.assertTrue(torch.allclose(logits_merged, logits_merged_from_pretrained, atol=atol, rtol=rtol))
 
+    def _test_offload_merge(self):
```

Comment:
Note that this function is not being called right now. It needs to be called from a top class that is a `unittest.TestCase`, like the other tests in this class. And since it requires a GPU, it should be moved to `test_gpu_examples.py`.

Reply:
Test has been moved and is called from a class that inherits from `unittest.TestCase`